doc/whats_new.rst

```diff
@@ -119,6 +119,16 @@ Bug fixes
      the same result as the LassoLars implementation available
      in R (lars library). :issue:`7849` by `Jair Montoya Martinez`_
 
+API changes summary
+-------------------
+
+   - Ensure that estimators' attributes ending with ``_`` are not set
+     in the constructor but only in the ``fit`` method. Most notably
+     if you implemented your own estimator deriving from
+     :class:`ensemble.BaseEnsemble` you may be affected by this if you
+     are using ``self.estimators_`` in your constructor.
```

Comment:
or even in `fit` or even in random user code. This can also be true of someone selecting trees from a forest and putting them together into a new ensemble by extending a clean forest...